spec/board_spec.rb

```diff
@@ -36,4 +36,21 @@
     end
   end
 
+  describe '#valid_placement?' do
+    
+    it 'checks if the placement is equal to length of ship' do
```

Comment:
I feel like we should have a test for a case where this is true as well? Or just make sure other cases of this are accounted for?

Reply:
Yeah, we'll make sure to add it in as we write tests for the rest of this method.